front/src/applications/operationalStudies/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx

```diff
@@ -91,28 +91,21 @@ export default function ImportTrainScheduleConfig({
         return true;
       }
       const hasInvalidteps = trainSchedule.steps.some((step) =>
```

Comment:
Oups, typo, could you correct it please ? :pray: 
```suggestion
      const hasInvalidSteps = trainSchedule.steps.some((step) =>
```

Reply:
Yes, corrected ;)